lxd/images.go

```diff
@@ -10,11 +10,12 @@ import (
 	"os"
 	"os/exec"
 	"strconv"
-
```

Comment:
Can we keep this newline? We're using this to delineate visually between imports that are in the stdlib, and our own.